docs/farming-&-staking/staking/operators/tips-operator.mdx

```diff
@@ -18,7 +18,7 @@ An operator needs a key pair to participate in bundle production.
 You can create a key using the following command:
 
 ```bash
-target/production/subspace-node key generate --scheme sr25519
+target/production/subspace-node run key generate --scheme sr25519
```

Comment:
@nazar-pc should this work? When I try to `./subspace-node run key ...` I get an `error: unexpected argument 'key' found`.

Reply:
No, `run x` doesn't make sense, that command is for running exclusively. All key-related commands are gone except importing, but now I see that we need to bring generation back. I'll start a Slack thread about this to figure out what the interface should look like,